crates/tauri/src/manager/webview.rs

```diff
@@ -680,14 +680,14 @@ fn on_webview_event<R: Runtime>(webview: &Webview<R>, event: &WebviewEvent) -> c
           paths: Some(paths),
           position,
         };
-        webview.emit_to_webview(DRAG_ENTER_EVENT, payload)?
+        webview.emit_to_webview(&DRAG_ENTER_EVENT, payload)?
```

Comment:
isn't `DRAG_ENTER_EVENT` a copy type? can't we remove the `&`?

Reply:
Was indicative of forgetting to make `emit_to_webview` also take `EventName`. The `&event` is sort of magic to get Rust to deref coerce `&EventName<&str>` to `&str`. Since this is an [unidiomatic use](https://rust-lang.github.io/api-guidelines/predictability.html#c-deref) of `Deref`, I've removed the trait impl in the newest version.

Reply:
I didn't mean to deref, I meant that `DRAG_ENTER_EVENT` is of type `EventName<&str>` so it should be copy and so we can have `emit_to_webview` method take `EventName<&str>` and use this const directly without `&`.